README.md

```diff
@@ -33,6 +33,12 @@ HEADLESS=false npm run test:local
 
 If you want to take screenshots of the browser as the tests run, you can add `RECORD=true` as an environment variable. The screenshots go in a unique folder within the `screenshots` directory. If your tests fail it will print out which folder the screenshots for that specific test are in.
 
+### Running the tests remotely (on the actual e2e environment)
+
+Connect to the e2e vpn
+
+Then run the codebuild job `apply-dev-sgs-to-e2e-test` in `bichard7-shared`, then run your tests and remember to run `remove-dev-sgs-from-e2e-test` when you are finished.
+
 ##Environment Variables
```

Comment:
Not related to this PR, but can we add a white space before `Environment` so the header is rendered correctly?